packages/babel-config/src/plugins/babel-plugin-redwood-routes-auto-loader.ts

```diff
@@ -43,6 +43,26 @@ export default function (
   // @NOTE: This var gets mutated inside the visitors
   let pages = processPagesDir().map(withRelativeImports)
 
+  // Currently processPagesDir() can return duplicate entries when there are multiple files
+  // ending in Page in the individual page directories. This will cause an error upstream.
+  // Here we check for duplicates and throw a more helpful error message.
+  const duplicatePageImports = new Set<string>()
+  const sortedPageImports = pages.map((page) => page.importName).sort()
```

Comment:
Minor suggesiton to improve the naming of this variable here cause it's not import statements its just an array of strings like `['AboutPage', 'AboutPage', 'FatalErrorPage']`

```suggestion
  const sortedPageImportNames = pages.map((page) => page.importName).sort()
```

Reply:
Agreed. Updating in ce11673213a6b1578e3291182b463e1e4824af08